src/core_atmosphere/diagnostics/soundings.F

```diff
@@ -205,8 +205,8 @@ subroutine soundings_compute()
 
         use mpas_derived_types, only : MPAS_pool_type
         use mpas_pool_routines, only : MPAS_pool_get_dimension, MPAS_pool_get_array
-        use mpas_timekeeping, only : MPAS_time_type, MPAS_is_alarm_ringing, MPAS_reset_clock_alarm, MPAS_get_clock_time, &
-                                     MPAS_get_time, MPAS_NOW
+        use mpas_derived_types, only : MPAS_Time_type, MPAS_NOW
+        use mpas_timekeeping, only : MPAS_is_alarm_ringing, MPAS_reset_clock_alarm, MPAS_get_clock_time, MPAS_get_time 
```

Comment:
Let's delete the extra space a the end of this line.

Reply:
@mgduda I just forced push a change that removes the extra space at line 209! Thanks for catching it!